config/action.d/mail-whois-lines.conf

```diff
@@ -4,6 +4,10 @@
 # Modified-By: Yaroslav Halchenko to include grepping on IP over log files
 #
 
+[INCLUDES]
+
+before = mail-whois-common.conf
```

Comment:
-autodetect and -basic sounds reasonable (and consistent to the way existing .conf files are structured).

For the moment, I did not default to -autodetect because of new dependencies (file, iconv). That said, please tell me if you still want to default to -autodetect.